### PR TITLE
Suggested addition of run_master.sh example

### PR DESCRIPTION
**Pull request by wmwv**
Proposal for addition of specific run-one-test example in README.

> If you are submitting a PR for a validation test (either adding a new one or updating an existing one), please post a link to the DESCQA web interface for the test you are submitting.

### CODE REVIEW

**Comment by wmwv** [comment]
OK, instead of this, perhaps a link on the README.md to suggest to go look at CONTRIBUTING.md to learn how to run a test.

File: README.md [hunk 1]
```diff
@@ -11,8 +11,19 @@ A [web interface](https://portal.nersc.gov/cfs/lsst/descqa/) hosted on NERSC dis
...
-## Quick start guide
+To run a sample DESCQA test from the command-line:
+
+```
+git clone git@github.com:LSSTDESC/descqa
+cd descqa
+sh run_master.sh -c cosmoDC2_v1.1.4_small -t SkyArea
+```
+
+If you are in an environment that has GCRCatalogs installed along with, such as `desc-python` at NERSC
```
Comment (yymao) [inline README.md:24]:
```suggestion
See ["How to run DESCQA?" section in `CONTRIBUTING.md`](.CONTRIBUTING.md#how-to-run-descqa) for more options that you can use with the run script. 
```

**Review by yymao — CHANGES_REQUESTED** [review]
The run script should start the environment for you.

File: README.md (hunk 1 above)
Comment (wmwv) [reply README.md:24]: > The run script should start the environment for you.

Oh, awesome, great.

**Review by yymao — APPROVED** [review]

**Pull request merged**